Sources/MapboxNavigation/OrnamentsController.swift

```diff
@@ -153,10 +154,10 @@ class OrnamentsController: NavigationComponent, NavigationComponentDelegate {
                                                                                             y: y - navigationView.safeAreaInsets.bottom)
         case .compact:
             if UIApplication.shared.statusBarOrientation == .landscapeRight {
-                navigationMapView.mapView.ornaments.options.attributionButton.margins = CGPoint(x: -navigationView.safeAreaInsets.right,
+                navigationMapView.mapView.ornaments.options.attributionButton.margins = CGPoint(x: x - navigationView.safeAreaInsets.right,
```

Comment:
Increase the margin distance to the same as the floating buttons. It would solve the issue where the button is sometimes untappable in landscape mode because too close to the margin.